src/libs/elektra/keyname.c

```diff
@@ -1229,21 +1276,28 @@ const char * keyBaseName (const Key * key)
 
 
 /**
- * Calculates number of bytes needed to store basename of @p key.
+ * Calculates number of bytes needed to store basename of @p key. (including
+ * NULL terminator)
```

Comment:
```suggestion
 * Calculates number of bytes needed to store basename of @p key (including NULL terminator).
```

Reply:
fixed in https://github.com/ElektraInitiative/libelektra/pull/3795/commits/cea2c469decb39e99b3d71197f27ed40dede38d9